js/layout-javascript/small-card-code.js

```diff
@@ -61,6 +65,9 @@ function DynamicList(id, data, container) {
 
   this.data.bookmarksEnabled = _this.data.social.bookmark;
 
+  // this is used for calling photoswipe library, do not remove
+  var photoswipeTemplate = Fliplet.Widget.Templates['templates/photoswipe.photoswipe'];
```

Comment:
In what way is this used?

`Fliplet.Navigate.previewImages()` has a similar line but it's being used within the function.

![image](https://user-images.githubusercontent.com/290733/90386058-e96c2780-e07b-11ea-8b0f-88c5ac7b50d7.png)

It doesn't look like `photoswipeTemplate` is being used at all.

Reply:
Actually you are right. Although before it did not work without it, maybe there was another mistake. I removed photoswipe vars.